pkg/virt-config/config-map.go

```diff
@@ -248,6 +257,10 @@ func setConfig(config *Config, configMap *k8sv1.ConfigMap) error {
 		config.MachineType = machineType
 	}
 
+	if cpuRequest := strings.TrimSpace(configMap.Data[cpuRequestKey]); cpuRequest != "" {
+		config.CPURequest = cpuRequest
```

Comment:
Can that compile? Shouldn't there be something like `Parse(cpuReuests)`?

Reply:
No :) the 'verified' flag is really missing here...